save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationContestsMenu.kt

```diff
@@ -101,6 +124,11 @@ external interface OrganizationContestsTableProps<D : Any> : TableProps<D> {
      * Flag to update table data when contest is created
      */
     var isContestCreated: Boolean
+
+    /**
+     * Fun update contest
+     */
+    var updateContest: (ContestDto) -> Unit
```

Comment:
```suggestion
    var deleteContest: (ContestDto) -> Unit
```
The callback from props is used only for deletion

Reply:
Renamed